### PR TITLE
feat(studio): adds guided tour button to top

### PR DESCRIPTION
**Pull request by ptrckbp**
Running a little experiment to see if people want to use the tutorial / help buttons in app. On pages with no help icon (like the flow editor), a tutorial button appears (see images), where the help button would normally be and triggers the Guided Tour. 

This also standardizes the buttons to have the same spacing, and icon + title. 

On a page with no existing help button :
Before
image

After
image

On a page with an existing help button :
Before
image

After
image

This also adds analytics to the tutorial steps and simplifies the tracking implementation by removing error handling.

### CODE REVIEW

**Comment by ptrckbp** [comment]
Also adds this step to the guided tour so people know to click to re-read :
image

File: packages/studio-ui/src/web/util/InjectSegment.ts [hunk 1]
```diff
@@ -48,17 +48,21 @@ export default (user: UserReducer): void => {
   })
 }
 
-function trackEvent(eventName: string, payload?: any, options?: any, callback?: (...params: any[]) => any) {
+function trackEvent(eventName: string, payload?: any, options?: any, callback?: (...params: any[]) => any): void {
   if (analytics) {
     // analytics only defined if window.SEND_USAGE_STATS is true
-    return analytics.track(eventName, payload, options, callback)
+    analytics.track(eventName, payload, options, callback).catch(e => {
+      console.error('Analytics error - event', e)
```
Comment (laurentlp) [inline packages/studio-ui/src/web/util/InjectSegment.ts:55]: Do we really want to log those errors? I don't think it has much value to let the user know that sending analytics failed. What do you think?

**Review by laurentlp — COMMENTED** [review]
Small comment otherwise LGTM! Nice work!

**Comment by laurentlp** [comment]
@ptrckbp Any update?

**Review by samuelmasse — APPROVED** [review]
LGTM

**Pull request merged**